aslprep/cli/workflow.py

```diff
@@ -190,4 +190,4 @@ def build_boilerplate(config_file, workflow):
         except (FileNotFoundError, CalledProcessError, TimeoutExpired):
             config.loggers.cli.warning("Could not generate CITATION.tex file:\n%s", " ".join(cmd))
         else:
-            copyfile(pkgrf("aslprep", "data/boilerplate.bib"), citation_files["bib"])
+            copyfile(load_data.readable("boilerplate.bib"), citation_files["bib"])
```

Comment:
Here you could use:

```python
with load_data.as_file("boilerplate.bib") as bib_file:
    copyfile(bib_file, citation_files["bib"])
```

This would create a context manager that ensures there's an unzipped file to copy and then clean up. However, since you've used `load_data()` on this file anyway, there's already a cached version that will be cleaned up at program exit, so:

```suggestion
            copyfile(load_data("boilerplate.bib"), citation_files["bib"])
```

Reply:
Thanks so much @effigies! I had no clue `data.load` could do so much.